iambic/core/models.py

```diff
@@ -685,8 +685,13 @@ def __getitem__(self, item):
 
 
 class ExpiryModel(IambicPydanticBaseModel):
-    expires_at: Optional[Union[str, datetime.datetime, datetime.date]] = Field(
-        None, description="The date and time the resource will be/was set to deleted."
+    expires_at: Optional[str] = Field(
```

Comment:
it's actually dangerous to remove date time.datetime, and date time.date because a model definition means programatic contraction accepts it being a a valid type.

When you remove daytime.datetime, and datetime.date, it means any API usage of expires_at_default using will not be valid anymore.

I will suggest restoring the original stanza.

Reply:
I attempt to deal with the string issue by modifying the upstream source code on this PR: https://github.com/noqdev/iambic/pull/621